src/configuration_parser.cpp

```diff
@@ -87,3 +97,9 @@ bool ConfigurationParser::ParseConfigFile(const std::string& path) {
   return true;
 }
 
+void ConfigurationParser::GenerateHelpMessage(char *argv[]) {
```

Comment:
Would be a bit more accurate:
```suggestion
void ConfigurationParser::PrintHelpMessage(char *argv[]) {
```